tempesta_fw/http_parser.c

```diff
@@ -1348,6 +1348,7 @@ __parse_transfer_encoding(TfwHttpMsg *hm, unsigned char *data, size_t len)
 /* Main (parent) HTTP request processing states. */
 enum {
 	Req_0,
+	Req_CRLF,
```

Comment:
In my opinion, this may appear too generic. Something like **Req_0_CRLF** may visibly clarify that this is one of the initial states.